includes/options/class-amp-options-menu.php

```diff
@@ -415,95 +403,6 @@ public function render_theme_support() {
 		<?php
 	}
 
-	/**
-	 * Post types support section renderer.
-	 *
-	 * @todo If dirty AMP is ever allowed (that is, post-processed documents which can be served with non-sanitized valdation errors), then automatically forcing sanitization in standard mode should be able to be turned off.
-	 *
-	 * @since 1.0
-	 */
-	public function render_validation_handling() {
-		?>
-		<fieldset>
-			<?php
-			$auto_sanitization = AMP_Validation_Error_Taxonomy::get_validation_error_sanitization(
-				[
-					'code' => 'non_existent',
-				]
-			);
-
-			$forced_sanitization = 'with_filter' === $auto_sanitization['forced'];
-			?>
-
-			<?php if ( $forced_sanitization ) : ?>
-				<div class="notice notice-info notice-alt inline">
-					<p><?php esc_html_e( 'Your install is configured via a theme or plugin to automatically sanitize any AMP validation error that is encountered.', 'amp' ); ?></p>
```

Comment:
@westonruter Is this information still needed? This comes from a different filter: `'amp_validation_error_sanitized'`.